index.Rmd

```diff
@@ -7,7 +7,7 @@ output:
     code_folding: show
 ---
 
-# Introduction
+# Introduction. 
```

Comment:
Would be great if you could cite your sources here since you reference literature. For example, how do you know the propensity for visiting urban parks is affected by attractiveness and convenience instead of other factors?